### PR TITLE
Change the custom error to Ballerina error

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
Change the custom error to Ballerina error

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ user secrets via SMS or voice message, verify OTP, and add and delete users. It
         var details = twilioClient->getAccountDetails();
         match details {
             twilio:Account account => io:println(account);
-            twilio:TwilioError twilioError => io:println(twilioError);
+            twilio:error twilioError => io:println(twilioError);
```
Comment (VijithaEkanayake) [inline README.md]:
```suggestion
            error twilioError => io:println(twilioError);
```

File: guide/twilio_sample.bal [hunk 2]
```diff
@@ -61,7 +61,7 @@ public type TwilioConnector object {
     public http:Client client;
 
     documentation { Return account details of the given account-sid
-        R{{}} If success, returns account object with basic details, else returns TwilioError object
+        R{{}} If success, returns account object with basic details, else returns error object
```
Comment (VijithaEkanayake) [inline guide/twilio_sample.bal]: Just say returns error without saying error object

File: README.md (hunk 1 above)
Comment (kalaiyarasiganeshalingam) [reply README.md]: Fixed

File: guide/twilio_sample.bal (hunk 2 above)
Comment (kalaiyarasiganeshalingam) [reply guide/twilio_sample.bal]: Fixed

**Pull request merged**